config/initializers/env.rb

```diff
@@ -40,3 +40,11 @@
   search_form_autocomplete: ENV['ENABLE_SEARCH_FORM_AUTOCOMPLETE'],
   entity_page: ENV['ENABLE_ENTITY_PAGE']
 )
+
+# Add locales to available locales
+if ENV['I18N_ADDITIONAL_LOCALES']
+  ENV['I18N_ADDITIONAL_LOCALES'].split(',').each do |locale|
+    Rails.application.config.i18n.available_locales << locale.to_sym
+  end
+  Rails.application.config.i18n.available_locales.sort!
```

Comment:
This might be a good place to remove duplicates, if they're part of the standard locales and added again via I18N_ADDITIONAL_LOCALES, or added multiple times through I18N_ADDITIONAL_LOCALES